### PR TITLE
Add TON support for Swap (and Exchange in general)

### PR DESCRIPTION
**Pull request by sprohaszka-ledger**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

~~Update Wallet-API core version to support TON serialization.~~
Update *partner's info test* format to be consistent with CAL format.
Add more integration test in AppExchange with dedicated TON payload.
Fix infinite loader issue when `DISABLE_BROADCAST` and use old swap UI.

### ❓ Context

- **JIRA or GitHub link**: [LIVE-13901] [LIVE-14520]

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-13901]: https://ledgerhq.atlassian.net/browse/LIVE-13901?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: turbo.json [hunk 1]
```diff
@@ -37,15 +40,17 @@
...
+    "NO_DEBUG_DEVICE",
     "SENTRY_AUTH_TOKEN",
     "SENTRY_ENVIRONMENT",
     "SENTRY_PROJECT",
     "SHORT_BUNDLE_ID",
-    "SUPPLY_UPLOAD_MAX_RETRIES"
+    "SUPPLY_UPLOAD_MAX_RETRIES",
+    "VERBOSE"
   ],
-  "ui": "tui",
```
Comment (Justkant) [inline turbo.json:48]: Did you have any issue with `tui` ? was it to debug properly ?

Comment (sprohaszka-ledger) [reply turbo.json:48]: Yes, I didn't liked it and move to regular console display. I will roll back this modification.

Comment (Justkant) [reply turbo.json:48]: Thx

File: libs/ledgerjs/packages/hw-app-exchange/src/Exchange.ts [hunk 2]
```diff
@@ -31,6 +31,8 @@ const PROCESS_TRANSACTION_RESPONSE = 0x06;
 const CHECK_TRANSACTION_SIGNATURE = 0x07;
 const CHECK_PAYOUT_ADDRESS = 0x08;
 const CHECK_REFUND_ADDRESS = 0x09;
+// const CHECK_REFUND_ADDRESS_AND_DISPLAY = 0x09;
+// const CHECK_REFUND_ADDRESS_NO_DISPLAY = 0x0c;
```
Comment (Justkant) [inline libs/ledgerjs/packages/hw-app-exchange/src/Exchange.ts:35]: Should it be remove ?

**Review by Justkant — COMMENTED** [review]
Just the question for the commented constants
We need to redo the `pnpm i` with a fresh lock file from develop
And add a changeset and we're good

File: libs/ledgerjs/packages/hw-app-exchange/src/Exchange.ts (hunk 2 above)
Comment (sprohaszka-ledger) [reply libs/ledgerjs/packages/hw-app-exchange/src/Exchange.ts:35]: Not yet as they will be used in future commit. This are new APDUs to use with Exchange flow.

**Review by Justkant — APPROVED** [review]

**Pull request merged**